include/umf/memory_pool_ops.h

```diff
@@ -163,7 +163,7 @@ typedef struct umf_memory_pool_ops_t {
     ///   otherwise the pool's name is returned.
     /// @return UMF_RESULT_SUCCESS on success or appropriate error code on failure.
     ///
-    umf_result_t (*ext_get_name)(void *pool, const char **name);
+    umf_result_t (*get_name)(void *pool, const char **name);
```

Comment:
I get that you updated the comment above, but shouldn't this function be now moved above the `The following function is optional...` section?

Reply:
done